clouddriver-kubernetes/src/main/groovy/com/netflix/spinnaker/clouddriver/kubernetes/v2/description/manifest/KubernetesManifestAnnotater.java

```diff
@@ -228,6 +234,21 @@ public static KubernetesCachingProperties getCachingProperties(KubernetesManifes
         .build();
   }
 
+  public static void useSourceCapacity(KubernetesHandler deployer, KubernetesManifest manifest, KubernetesV2Credentials credentials) {
+    Map<String, String> annotations = manifest.getAnnotations();
+    if (deployer instanceof CanScale &&
+        annotations.containsKey(USE_SOURCE_CAPACITY) &&
+        annotations.get(USE_SOURCE_CAPACITY).equals("true")) {
+      KubernetesManifest deployment = credentials.get(manifest.getKind(), manifest.getNamespace(), manifest.getName());
```

Comment:
Could you rename this var from `deployment` to `manifest` since this might not be a deployment?